lib/matplotlib/path.py

```diff
@@ -325,11 +325,11 @@ def make_compound_path_from_polys(cls, XY):
 
     @classmethod
     def make_compound_path(cls, *args):
-        """Make a compound path from a list of Path objects."""
+        """Make a compound path from a list of Path objects. Blindly removes
+        all Path.STOP control points."""
```

Comment:
```suggestion
        """
        Make a compound path from a list of Path objects. Blindly removes all
        Path.STOP control points.
        """
```
We put the quotes on separate lines if there is more than one row. (just noted that this is not documented in our docstring guide ...)